urdf/include/common_macros.urdf.xacro

```diff
@@ -266,13 +334,40 @@
       </visual>
     </xacro:if>  
     <collision>
-      <origin xyz="0 0 0.09" rpy="0 0 0"/>
+      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
       <geometry>
         <box size="${base_prop['base']['x_size']} ${base_prop['base']['y_size']} ${base_prop['base']['z_size']}"/>
       </geometry>
     </collision>
     </link>
-  </xacro:macro> 
+  </xacro:macro>
+
+<!-- ===================== Footprint xacro ==========================================
+
+  Xacro to create the base link. For more detail information and usage, see:
+    https://github.com/pxalcantara/mobile_robot_description/wiki/Base-xacro
+
+  params:
+  - base_props [dictionary]: base link properties;
+  - mesh [string]: file name of the link mesh;
+    - DEFAULT Value -> Empty string  
+  - origin [block]: origin, position and orientation of the link;
+-->
+  <xacro:macro name="footprint"
+               params="wheel_props">
```

Comment:
I don't think we're using the `wheel_props` parameter right? Please, can we remove it and also update the xacro documentation?